doc/examples/tracking_tissue_classifier.py

```diff
@@ -0,0 +1,319 @@
+streamline stops in a position classified as 'ENDPOINT' or 'OUTSIDEIMAGE'. A
+streamline is 'invalid' when it stops in a position classified as 'TRACKPOINT'
+or 'INVALIDPOINT'. These conditions are described below. The 'LocalTracking'
+generator can be set to output all generated streamlines or only the 'valid'
+ones.
+
+This example is an extension of the
+:ref:``example_deterministic_fiber_tracking`` example. We begin by loading the
+data, fitting a constant solid angle (CSA) reconstruction
+model and creating the maximum determnistic direction getter.
+"""
+
+import numpy as np
+
+from dipy.data import read_stanford_labels, default_sphere
+from dipy.direction import DeterministicMaximumDirectionGetter
+from dipy.io.trackvis import save_trk
+from dipy.reconst.shm import CsaOdfModel
+from dipy.tracking.local import LocalTracking
+from dipy.tracking import utils
+from dipy.viz import fvtk
+from dipy.viz.colormap import line_colors
+
+ren = fvtk.ren()
+
+hardi_img, gtab, labels_img = read_stanford_labels()
+data = hardi_img.get_data()
+labels = labels_img.get_data()
+affine = hardi_img.get_affine()
+
+seed_mask = labels == 2
+white_matter = (labels == 1) | (labels == 2)
+seeds = utils.seeds_from_mask(seed_mask, density=2, affine=affine)
+
+csa_model = CsaOdfModel(gtab, 4)
+csa_fit = csa_model.fit(data, mask=white_matter)
+
+dg = DeterministicMaximumDirectionGetter.from_shcoeff(csa_fit.shm_coeff,
+                                                      max_angle=30.,
+                                                      sphere=default_sphere)
+
+"""
+Threshold Tissue Classifier
+---------------------------
+A scalar map can be used to build a tissue classifier by thresholding it to a
+fixed value. Here, we show an example using the fractional anisotropy (FA) map
+of the DTI model. The threshold tissue classifier uses a trilinear
+interpolation method at the tracking position.
+
+**Parameters**
+
+- metric_map: numpy array [:, :, :]
+- threshold: float
+
+**Stopping criterion**
+
+- 'ENDPOINT': metric_map < threshold,
+- 'OUTSIDEIMAGE': tracking point outside of metric_map,
+- 'TRACKPOINT': stopped because no direction are available,
+- 'INVALIDPOINT': N/A.
+"""
+
+import matplotlib.pyplot as plt
+import dipy.reconst.dti as dti
+from dipy.reconst.dti import fractional_anisotropy
+from dipy.tracking.local import ThresholdTissueClassifier
+
+tensor_model = dti.TensorModel(gtab)
+tenfit = tensor_model.fit(data, mask=labels > 0)
+FA = fractional_anisotropy(tenfit.evals)
+
+threshold_classifier = ThresholdTissueClassifier(FA, .2)
+
+fig = plt.figure()
+mask_fa = FA.copy()
+mask_fa[mask_fa < 0.2] = 0
+plt.xticks([])
+plt.yticks([])
+plt.imshow(mask_fa[:, :, data.shape[2] / 2].T, cmap='gray', origin='lower',
+           interpolation='nearest')
+fig.tight_layout()
+fig.savefig('threshold_fa.png')
+
+"""
+.. figure:: threshold_fa.png
+ :align: center
+
+ **Thresholded fractional anisotropy map.**
```

Comment:
Hey @gabknight  have you tried to render the tutorial and it see if it renders correctly?